cmake/Modules/FindRapidJson.cmake

```diff
@@ -26,4 +26,4 @@ if (NOT RAPID_JSON_INCLUDE_DIR OR NOT RAPID_JSON_LIBRARIES)
 endif ()
 
 include(FindPackageHandleStandardArgs)
-find_package_handle_standard_args(RAPID_JSON DEFAULT_MSG RAPID_JSON_INCLUDE_DIR )
\ No newline at end of file
+find_package_handle_standard_args(RapidJson DEFAULT_MSG RAPID_JSON_INCLUDE_DIR )
```

Comment:
Agh! Forgot to mention this change. It's a good idea to use the package name here spelled and cased the same way it is in the fileName just to make sure there's no confusion. Typically you expect `FindRapidJson` to populate `RapidJson_FOUND` not `RAPID_JSON_FOUND` for example.

Compiler warning:
```
CMake Warning (dev) at D:/Programming/CLion 2023.2.2/bin/cmake/win/x64/share/cmake-3.26/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args` (RAPID_JSON)
  does not match the name of the calling package (RapidJson).  This can lead
  to problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
  ```